src/sdk/keypair.ts

```diff
@@ -0,0 +1,69 @@
+import { toHexString } from '../utils';
+import {
+  cryptobox_keygen,
+  cryptobox_sk_to_u8vec,
+  cryptobox_pk_to_u8vec,
+  cryptobox_get_pk,
+} from 'node-kms';
+
+export type EIP712Type = { name: string; type: string };
+
+export type EIP712 = {
+  domain: {
+    chainId: number;
+    name: string;
+    verifyingContract: string;
+    version: string;
+  };
+  message: {
+    [key: string]: any;
+  };
+  primaryType: string;
+  types: {
+    [key: string]: EIP712Type[];
+  };
+};
+
+export const createEIP712 =
```

Comment:
I think this function is not up-to-date, right?

Reply:
I think it is missing a new variant for the delegated reencryption. It is not clearly specifed there https://github.com/zama-ai/tech-spec/tree/main/coprocessor#delegation-and-account-abstraction but I think that in this case we need to add an additional field for the delegatedAccount, wdyt? @immortal-tofu

Reply:
The reencryption is done yet. I'll review that 👍